firmware/application/apps/ui_fileman.cpp

```diff
@@ -614,6 +621,13 @@ FileManagerView::FileManagerView(
         } else
             nav_.display_modal("Open in Notepad", "Can't open that in Notepad.");
     };
+
+    button_rename_timestamp.on_select = [this]() {
+        if (selected_is_valid() && !get_selected_entry().is_directory) {
+            on_rename(::truncate(to_string_timestamp(rtc_time::now()), 8));
```

Comment:
When renaming an old log file, I was just thinking if I'd rather use the file's time stamp (last date it was written), versus the current date.  Just a thought.

Reply:
Interesting idea. I don't have a strong opinion either way.

Reply:
Me neither, but just thought I'd throw it out there.  :-)